admin/app/jobs/RebuildIndexJob.scala

```diff
@@ -13,7 +13,6 @@ class RebuildIndexJob(contentApiClient: ContentApiClient)(implicit executionCont
     extends GuLogging {
 
   val contentApiTagsEnumerator = new ContentApiTagsEnumerator(contentApiClient)
-  val tagPages = new TagPages
```

Comment:
No need to construct a new `TagPages` object, as its methods are all now static! This does mean we have to reference it by class name, rather than instance name, so we see a lot of `tagPages` becoming `TagPages` below.